smallrye-reactive-messaging-amqp/src/main/java/io/smallrye/reactive/messaging/amqp/AmqpConnector.java

```diff
@@ -133,7 +123,7 @@ void setup(ExecutionHolder executionHolder) {
 
         AmqpClient client = AmqpClientHelper.createClient(this, ic, clientOptions);
         String link = ic.getLinkName().orElseGet(ic::getChannel);
-        ConnectionHolder holder = new ConnectionHolder(client, ic);
+        ConnectionHolder holder = new ConnectionHolder(client, ic, getVertx());
```

Comment:
Should we be passing Vertx or the ExecutionHolder?

Reply:
Here, we only need the vert.x instance. But maybe later we will need more.